app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -79,6 +79,32 @@ def metrics_connect
                                      authentication_password('default'))
     end
 
+    def run_generic_operation(operation, path)
+      require 'hawkular_all'
+      host_port = "#{hostname}:#{port}"
```

Comment:
Will this work with IPv6?  Probably should use the URI class to construct it properly.